app/src/main/java/space/taran/arknavigator/mvp/model/backup/ArkBackup.kt

```diff
@@ -0,0 +1,78 @@
+package space.taran.arknavigator.mvp.model.backup
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import space.taran.arknavigator.mvp.model.ArkFiles
+import space.taran.arknavigator.mvp.model.repo.FoldersRepo
+import java.nio.file.Path
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+import java.util.zip.ZipEntry
+import java.util.zip.ZipOutputStream
+import kotlin.io.path.Path
+import kotlin.io.path.createDirectories
+import kotlin.io.path.exists
+import kotlin.io.path.inputStream
+import kotlin.io.path.outputStream
+
+class ArkBackup(private val foldersRepo: FoldersRepo) {
```

Comment:
Maybe `StorageBackup`?

Reply:
`ArkBackup` also backup file with favorites. There may be other files in future.

Reply:
Actually "storage" is everything what is replicated/persisted here. We have favorites storage, tags storage and index.

Reply:
Changed to `StorageBackup`